core/src/test/java/com/radixdlt/rev2/RustMempoolTest.java

```diff
@@ -383,6 +302,51 @@ public void test_rust_mempool_getRelayTxns() throws Exception {
     }
   }
 
+  private NodeRustEnvironment createNodeRustEnvironment() throws IOException {
```

Comment:
I wonder if it's worth moving this to some unit test helper or something? Given that we've duplicated it across a few tests now?

Reply:
With the tests, we've been following the "Good Scout" rule and leaving our Java tests in a better shape than we found them every change.

Reply:
Reworked tests to reuse common code. Unfortunately, due to Gradle issues it appeared too time-consuming to attempt to reuse test dependencies, and I ought to create two similar (but not identical) base classes for tests which require NodeRustEnvironment.

Reply:
This is what `core/src/test-core` is for :). Create a new file `NodeRustEnvironmentBuilder.java` in `core/src/test-core/java/com/radixdlt/rev2`.

We've agreed as a team to use composition over inheritance for any new test utilities, so please don't introduce base classes :)

Reply:
Such a location does not allow using it from `core-rust-bridge`, where 3 tests are located.

Reply:
Reworked to have it as a utility class, rather than a base class.